src/diff_opt.jl

```diff
@@ -189,42 +187,39 @@ MOI.set(model, DiffOpt.BackwardInVariablePrimal(), x)
 """
 struct BackwardInVariablePrimal <: MOI.AbstractVariableAttribute end
 
+"""
+    BackwardOutObjective
+
+A `MOI.AbstractModelAttribute` to get output data to backward differentiation,
+that is, problem input data.
+
+```julia
+MOI.get(model, DiffOpt.BackwardOutObjective)
+```
```

Comment:
it would be great to have two line showing:
- get sensitivity of linear part of obj
- get sensitivity of quad part of obj

Reply:
Done